components/LevitatingLink.js

```diff
@@ -1,7 +1,8 @@
-export default function LevitatingLink ({ href, children }) {
+export default function LevitatingLink ({ href, children, handleClick, active }) {
```

Comment:
I think adding this functionality shouldn’t need to cause this component to change. We should be detecting the active menu item based on scroll.